tests/tape_tests/test_ensemble.py

```diff
@@ -168,6 +179,50 @@ def test_from_qso_dataset(dask_client):
     assert res.loc[1257836]["z"] == pytest.approx(53.013018, rel=0.001)
 
 
+def test_read_rrl_dataset(dask_client):
+    """
+    Test a basic load and analyze workflow from the S82 RR Lyrae Dataset
+    """
+
+    ens = tape.read_dataset("s82_rrlyrae", dask_client=dask_client)
+
+    # larger dataset, let's just use a subset of ~100
```

Comment:
Is it 100 or 350?

Reply:
Looks like this was a repeated copy and paste error. I've cleaned those up (and removed the comment reference to the subset size :).